client/src/components/Workflow/WorkflowList.vue

```diff
@@ -37,11 +37,10 @@
                     @onRestore="onRestore" />
             </template>
             <template v-slot:cell(tags)="row">
-                <Tags
-                    :index="row.index"
-                    :tags="row.item.tags"
+                <StatelessTags
+                    :value="row.item.tags"
```

Comment:
```suggestion
                <StatelessTags
                    clickable
                    :value="row.item.tags"
```

Adds clickability visuals to the filters.